homeassistant/components/caldav/coordinator.py

```diff
@@ -76,6 +76,7 @@ async def async_get_events(
                 continue
             event_list.append(
                 CalendarEvent(
+                    uid=get_attr_value(vevent, "uid"),
```

Comment:
Is this still useful or used for anything? I see in the test it just returns "0" which i don't think we want if this doesn't have a valid uid.